### PR TITLE
Clean up and fix some issues in function examples

### PR DESCRIPTION
**Pull request by kars7e**
* Fix functions to work properly with cloudmaster
* Organize files by types
* Tune README to the current content (more work needed)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,189 +192,34 @@ Since we're done with this, let's delete our API endpoint we are going to assign
...
 ```
-$ dispatch get subscriptions
-           NAME          | SOURCE TYPE |   EVENT TYPE   | FUNCTION NAME | STATUS |         CREATED DATE
-----------------------------------------------------------------------------------------------------------
-  native-troll-352185    | vcenter     | vm.powered.on  | status        | READY  | Sat Jan  1 12:48:45 PST 0000
-  willing-cheetah-356512 | vcenter     | vm.removed     | status        | READY  | Sat Jan  1 12:48:44 PST 0000
-  touched-foal-822640    | vcenter     | vm.powered.off | status        | READY  | Sat Jan  1 12:48:46 PST 0000
-  just-wildcat-236741    | vcenter     | vm.stopping    | status        | READY  | Sat Jan  1 12:48:44 PST 0000
-  star-goose-402131      | vcenter     | vm.deployed    | status        | READY  | Sat Jan  1 12:48:39 PST 0000
+certbot certonly --standalone -d ato-api.dispatchframework.io
```
Comment (berndtj) [inline README.md]: maybe we should use more generic DNS names `api.example.com` for instance

File: all-resources.yaml [hunk 2]
```diff
@@ -0,0 +1,200 @@
+# Before running this file, create required secrets. See README.md for details
```
Comment (berndtj) [inline all-resources.yaml:1]: I would probably assume that the user has already ran `dispatch create seed-images`.  If not we should point folks to that to create the generic images, and add on top of them the stuff for the demo.

File: apis/apis.yaml [hunk 3]
```diff
@@ -11,4 +11,4 @@ uris:
   - /cloudmaster
 tags:
   - key: role
-    value : radio
\ No newline at end of file
+    value : ato
```
Comment (berndtj) [inline apis/apis.yaml]: Maybe we should just remove the tags :) or use generic tags.  I know it's my fault.

File: apis/ui-apis.yaml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+name: get-aws
+enabled: true
+function: aws
+cors: true
+methods:
+  - GET
+protocols:
+  - https
+hosts:
+  - ato-api.dispatchframework.io
```
Comment (berndtj) [inline apis/ui-apis.yaml]: We could just remove the hosts.  The API will use the org as a path based namespace

File: README.md (hunk 1 above)
Comment (kars7e) [reply README.md]: changed to generic DNS name

File: all-resources.yaml (hunk 2 above)
Comment (kars7e) [reply all-resources.yaml:1]: sounds good, I added instructions to use seed-images and changed the resource file.

File: apis/ui-apis.yaml (hunk 4 above)
Comment (kars7e) [reply apis/ui-apis.yaml]: went with a generic hostname

File: apis/apis.yaml (hunk 3 above)
Comment (kars7e) [reply apis/apis.yaml]: I removed them from all resources file, but forgot about others. Fixed :-)

**Pull request merged**